### PR TITLE
docs(deviceauth): migration to OpenAPI3

### PR DESCRIPTION
**Pull request by merlin-northern**
Changelog: None
Ticket: MEN-7714

### CODE REVIEW

File: backend/services/deviceauth/docs/devices_api.yml [hunk 1]
```diff
@@ -1,35 +1,18 @@
-swagger: '2.0'
+openapi: 3.0.0
```
Comment (tranchitella) [inline backend/services/deviceauth/docs/devices_api.yml:1]: How did you convert the specs? I suppose one of the (implicit) acceptance criteria was having an automated way to migrate from 2 to 3, so we can apply the same technique to the other repos.

Comment (merlin-northern) [reply backend/services/deviceauth/docs/devices_api.yml:1]: wooops I forgot about the research path. okey, this is done by hand and some scripting that is not even saved. I will do the research separately. sorry for that.

Comment (merlin-northern) [reply backend/services/deviceauth/docs/devices_api.yml:1]: no wait, I did some. but jira did not show me the whole ticket description! let me comment in the ticket.

Comment (merlin-northern) [reply backend/services/deviceauth/docs/devices_api.yml:1]: @tranchitella btw the ticket does not mention the research on the tools that can do the migration.

**Comment by merlin-northern** [comment]
@kjaskiewiczz I need an early review on this one, there are lots of things still to refactor in those tests, but I had to draw a line somewhere.

**Review by kjaskiewiczz — APPROVED** [review]

**Pull request merged**